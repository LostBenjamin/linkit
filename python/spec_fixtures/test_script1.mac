# test script

$max 10

[init]
red
<max>:rep
(render)

[doit 1]
blu

[render]
flu

