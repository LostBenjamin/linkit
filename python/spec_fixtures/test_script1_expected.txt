1:set:blu
10:set:red:10:rep:11:run
11:set:flu

